packages/create-toolpad-app/src/packageType.ts

```diff
@@ -0,0 +1,667 @@
+declare namespace PackageJsonTypes {
```

Comment:
Looks like this was copied from https://www.npmjs.com/package/type-fest. Shall we just add this as a dependency?

Reply:
I had done it to install this library before.
But there is a lot of code we didn't use it.
if we need to get more declarations from this library, we can change to that library.

Reply:
in spite of it not being important, it will waste our time installing dependency when it’s the first time to contribute.